github/util.py

```diff
@@ -221,3 +222,82 @@ def retrieve_email(username: str):
         )
     )
 
+def _create_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str
+):
+    # passed GitHub object must have org. admin authorization to create a team
+    organization = github.organization(organization_name)
+    team_list = list(filter(lambda t: t.name == team_name, organization.teams()))
+    if team_list:
+        util.verbose("Team {name} already exist".format(name=team_name))
+        return
+
+    try:
+        organization.create_team(name=team_name)
+        util.info("Team {name} created".format(name=team_name))
+    except ForbiddenError as err:
+        util.fail("{msg} Cannot create team {name} in org {org} due to missing privileges".format(
```

Comment:
why not name 'msg' 'err' - this would make it easier to spot that the exception is printed out =)

Reply:
Done